Release/tests/functional/websockets/client/send_msg_tests.cpp

```diff
@@ -456,6 +471,25 @@ TEST_FIXTURE(uri_address, send_stream_binary_msg_no_length)
     client.close().wait();
 }
 
+// Send an unsolicited pong message to the server
+TEST_FIXTURE(uri_address, send_pong_msg)
```

Comment:
I am happy adding the ifdef myself, but I am also happy if you do that :) Let me know how you want me to proceed.